### PR TITLE
[190] Research Programming Languages 

### PR DESCRIPTION
**Pull request by OluwaboriShobayo**

### CODE REVIEW

**Review by Ireti203 — COMMENTED** [review]
Why did you include the html file for the product page in this PR?

File: submissions/oluwaboriShobayo/html-forms/contactForm.html [hunk 1]
```diff
@@ -10,8 +10,12 @@ <h1>Style By Bori</h1>
         <a href="#">Home</a><br>
         <a href="#">Categories</a><br>
         <a href="#">About us</a><br>
+<<<<<<< HEAD
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/html-forms/contactForm.html:13]: Format here properly.

File: submissions/oluwaboriShobayo/html-forms/contactForm.html [hunk 2]
```diff
@@ -10,8 +10,12 @@ <h1>Style By Bori</h1>
...
+        <a href="productPage.html">Products</a>
+=======
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/html-forms/contactForm.html:15]: And here too

File: submissions/oluwaboriShobayo/html-forms/contactForm.html [hunk 3]
```diff
@@ -10,8 +10,12 @@ <h1>Style By Bori</h1>
...
         <a href="../HTML/productPage.html">Products</a><br>
         <a href="../html-forms/contactForm.html">Contact Us</a>
+>>>>>>> a10c2f42194fb95960a0957007c20a1fbc289106
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/html-forms/contactForm.html:18]: Same here

File: submissions/oluwaboriShobayo/html-forms/contactForm.html [hunk 4]
```diff
@@ -38,7 +42,11 @@ <h1>Contact Form</h1>
          <!--This section is for typing the message-->
             <label for="Message">Message:</label><br>
            <textarea name="message" id="message" cols="50" rows="10"></textarea><br><br>
+<<<<<<< HEAD
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/html-forms/contactForm.html:45]: Same here

File: submissions/oluwaboriShobayo/html-forms/contactForm.html [hunk 5]
```diff
@@ -38,7 +42,11 @@ <h1>Contact Form</h1>
...
+         <!--This houses the submit button-->
+=======
          <!--This section is for the submit button-->
+>>>>>>> a10c2f42194fb95960a0957007c20a1fbc289106
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/html-forms/contactForm.html:49]: And here

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/oluwaboriShobayo/HTML/productPage.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/oluwaboriShobayo/HTML/productPage.html:1]: Why are you creating a product page again?

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by OluwaboriShobayo** [comment]
I'm sorry, It was a mistake o

**Comment by Ireti203** [comment]
Kindly resolve the conflicts on this branch.

**Pull request closed without merging**